tests/main.js

```diff
@@ -422,6 +422,8 @@ testBundlers(
     t.deepEqual(
       functions,
       [
+        { name: 'test', mainFile: 'test.js', runtime: 'js', extension: '.js', srcFile: 'test.js' },
```

Comment:
I've updated this test because even though the listed function files are the same, the order in which they're listed has changed, which shouldn't matter for the purposes of `listFunctionFiles()`.